core/storage/statistics/gae_models_test.py

```diff
@@ -43,6 +46,125 @@ def test_create_and_get_event_models(self):
         self.assertEqual(event_model.is_feedback_useful, True)
 
 
+class StateCounterModelTests(test_utils.GenericTestBase):
+
+    def test_stat_model_gets_created(self):
+        model_instance = stat_models.StateCounterModel.get_or_create(
```

Comment:
For consistency, could you please replace stat_models with stats_models throughout? That's our usual alias for the module and it seems like it's only this file that uses stat_models.

Reply:
Done